x-pack/plugin/ml/qa/native-multi-node-tests/src/test/java/org/elasticsearch/xpack/ml/integration/RevertModelSnapshotIT.java

```diff
@@ -50,7 +62,15 @@ public void tearDownData() throws Exception {
         cleanUp();
     }
 
-    public void test() throws Exception {
+    public void testRevertModelSnapshot() throws Exception {
+        test(false);
+    }
+
+    public void testRevertModelSnapshot_DeleteInterveningResults() throws Exception {
+        test(true);
+    }
+
+    private void test(boolean deleteInterveningResults) throws Exception {
```

Comment:
It would be best if the job ID was passed as an argument and the two test cases set it to different values.  This will be helpful if a completely different test fails because of a side effect of one of these two tests.  (A long time ago almost all our integration tests used the same names, like `job` and `foo`, but this was a nightmare when a test failed because a job called `job` or `foo` was left over from an earlier test not completing as expected, so we tried to change all the integration test job names to be unique to a particular test.)

Reply:
Done.